hc/api/tests/test_ping.py

```diff
@@ -63,11 +66,37 @@ def test_it_reads_forwarded_protocol(self):
                             HTTP_X_FORWARDED_PROTO="https")
         ping = Ping.objects.latest("id")
         ### Assert the expected response status code and ping's scheme
+        self.assertEqual(r.status_code, 200)
+        self.assertEqual(ping.scheme, "https")
 
     def test_it_never_caches(self):
         r = self.client.get("/ping/%s/" % self.check.code)
         assert "no-cache" in r.get("Cache-Control")
 
     ### Test that when a ping is made a check with a paused status changes status
+    def test_paused_check_changes_status(self):
+        self.check.status = "paused"
+        self.check.save()
+
+        r = self.client.get("/ping/%s/" % self.check.code)
+        self.assertEqual(r.status_code, 200)
+
+        self.check.refresh_from_db()
+        self.assertEqual(self.check.status, "up")
     ### Test that a post to a ping works
+    def test_post_works(self):
```

Comment:
`test_post_ping_works` might be a better name.
`test_post_works` is a name that can be applied to testing the `post` method in general in which case it feels like you are testing the REST protocol

Reply:
Yeah, you're right George

Reply:
Worked on it.